note/技術/存檔.md

```
@@ -431,6 +450,7 @@ Player。Player 的 `character_id` 已透過 `user://saves/player_id.txt` 持久
 | [#170](https://github.com/adez360/Ailley/issues/170) | 記憶（L2/L4）接進角色存檔 —— **已完成** | #21 #22 記憶資料結構 |
 | [#342](https://github.com/adez360/Ailley/issues/342) | `character_library` 接進世界存檔 —— **已完成** | #21 |
 | [#343](https://github.com/adez360/Ailley/issues/343) | 開場讀檔：主選單「繼續遊戲」接上 `apply_world_save_data()`／`load_save_data()` —— **已完成** | #21 #22 #23 |
+| [#344](https://github.com/adez360/Ailley/issues/344) | 讀檔時重新生成場景裡缺席的角色 —— **已完成** | #21 #342 |
 
 #19 與 #20 沒有依賴 —— 兩則都是「存檔會用到但不屬於存檔」的東西，都做完了。
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)